docs/renovate.md

```diff
@@ -10,10 +10,13 @@ to catalog our packages and provide guidance to a developer who wants to test an
   packages in a single repository. This behaviour makes sense for us and helps reduce noise from Renovate PRs.
 * A lot of our JavaScript dev dependencies are provided by WordPress or WooCommerce globally at runtime, we include them
   as dev dependencies so that we aren't duplicating them in our build bundle but our unit tests can still pass by
-  having them available. This means we should keep the versions of these packages in the same range as our supported 
-  version of WordPress and WooCommerce.
+  having them available. This means we should keep the versions of these packages on the highest version avaiable in our
+  supported versions of WordPress and WooCommerce, giving us the best chance of catching any issues with the bundled
+  packages early.
 * Following on from above, we use the `@wordpress/dependency-extraction-webpack-plugin` to make WebPack aware of what
-  can be found globally at runtime. The configuration for this can be found in `webpack.config.js`.
+  can be found globally at runtime. The configuration for this can be found in `webpack.config.js`. Any `wordpress/*`
+  packages are bundled by default, in addition to any packages listed in the configuration file. Returning `null` in the
```

Comment:
> Any `wordpress/*`
>  packages are bundled by default, in addition to any packages listed in the configuration file.

I think it should be in a reversed way: these packages are not included in the bundle by default? 

Ref: paJDYF-2LK-p2#comment-8701 as well as your own internal summary here paJDYF-2LK-p2#comment-9078: 

> All wordpress/* packages are automatically removed from our build, so we rely on the globally available ones.

Reply:
Thanks for spotting that! You're quite right, updated in https://github.com/Automattic/woocommerce-payments/pull/3518/commits/0519f7202ae8d7afa782f1d602c63b5631616c64.